synolia/sylius-mail-tester-plugin/3.0/config/routes/sylius_mail_tester.yaml

```
@@ -0,0 +1,3 @@
+synolia_mail_tester:
```

Just a suggestion since imported route file is specifically `admin`

```suggestion
synolia_mail_tester_admin:
```

---

hi @diimpp, that's interesting but this plugin is not meant to add/alter a shop route so that's why i left like this